docs/example-config-files/terraform.bitops.config.yaml

```diff
@@ -1,8 +1,9 @@
 terraform:
     cli: 
       backend-config:
-        - KEY1=foo
-        - KEY2=bar
+        KEY1=foo
+        KEY2=bar
+      init-upgrade: true
```

Comment:
if the `--upgrade` flag is only used for `init` and not `apply`, i'd move this to the `options` section because having it here will also include it in `apply`.